### PR TITLE
fix(suite-native): fix not rendering text on android search in portfolio

### LINKED ISSUE #9474: Entered text in Search field not rendered.
**Describe the bug**
Entered text in Search field not rendered.
Works as expected on iOS

**Info:**

-   Mobile Suite Version: 23.9.1 (f15aff9)
-   OS: Android 12 and 13

**Screenshots**

https://github.com/trezor/trezor-suite/assets/31506317/6cec829b-ddfd-49d6-a00a-f119d4a0fea4

### PR DESCRIPTION
**Pull request by vytick**
## Description

search field on android did not render text input.

## Related Issue

Resolve #9474

### CODE REVIEW

File: suite-native/atoms/src/Input/SearchInput.tsx [hunk 1]
```diff
@@ -68,24 +66,30 @@ export const SearchInput = ({
...
+        onChange(value);
+    };
+
     return (
         <Pressable onPress={handleInputFocus}>
             <Box style={applyStyle(inputWrapperStyle, { isFocused, elevation })}>
                 <Icon name="search" color="iconSubdued" size="large" />
                 <TextInput
                     ref={searchInputRef}
-                    value={value}
```
Comment (Nodonisko) [inline suite-native/atoms/src/Input/SearchInput.tsx:87]: Why did you changed this to uncontrolled input?

**Review by Nodonisko — CHANGES_REQUESTED** [review]

File: suite-native/atoms/src/Input/SearchInput.tsx (hunk 1 above)
Comment (vytick) [reply suite-native/atoms/src/Input/SearchInput.tsx:87]: This was causing the text not to render. 

And previous implementation only make the value go round: it took the value, sent it to parent through `onChange()`, parent saved it, used length to decide if to show/hide the clear button and sent it back to the TextInput.

Comment (vytick) [reply suite-native/atoms/src/Input/SearchInput.tsx:87]: I've examined this yesterday and played a lot with it, but have not found a different solution, that would solve the issue

Comment (Nodonisko) [reply suite-native/atoms/src/Input/SearchInput.tsx:87]: That's very strange, it's basically standart controlled input which should work no matter what... but let's don't burn more time on it

**Review by Nodonisko — APPROVED** [review]

**Pull request merged**